### ISSUE TITLE
pip install fails

### ISSUE THREAD

**Issue by MarkEstefanos** [Ristellise/RedGalaxy#3]
running `pip install .` I get the following error:

```ERROR: Command errored out with exit status 1:
   command: /usr/bin/python3 /tmp/tmp6s9_fa0q get_requires_for_build_wheel /tmp/tmptw66vuzk
       cwd: /tmp/pip-req-build-yekcwz75
  Complete output (44 lines):
  configuration error: `project.dependencies[0]` must be pep508
  DESCRIPTION:
      Project dependency specification according to PEP 508

  GIVEN VALUE:
      "httpx >"

  OFFENDING RULE: 'format'

  DEFINITION:
      {
          "$id": "#/definitions/dependency",
          "title": "Dependency",
          "type": "string",
          "format": "pep508"
      }
  Traceback (most recent call last):
    File "/tmp/tmp6s9_fa0q", line 280, in <module>
      main()
    File "/tmp/tmp6s9_fa0q", line 263, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/tmp/tmp6s9_fa0q", line 114, in get_requires_for_build_wheel
      return hook(config_settings)
    File "/tmp/pip-build-env-j2aivp8_/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 338, in get_requires_for_build_wheel
      return self._get_build_requires(config_settings, requirements=['wheel'])
    File "/tmp/pip-build-env-j2aivp8_/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 320, in _get_build_requires
      self.run_setup()
    File "/tmp/pip-build-env-j2aivp8_/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 335, in run_setup
      exec(code, locals())
    File "<string>", line 1, in <module>
    File "/tmp/pip-build-env-j2aivp8_/overlay/lib/python3.8/site-packages/setuptools/__init__.py", line 108, in setup
      return distutils.core.setup(**attrs)
    File "/tmp/pip-build-env-j2aivp8_/overlay/lib/python3.8/site-packages/setuptools/_distutils/core.py", line 159, in setup
      dist.parse_config_files()
    File "/tmp/pip-build-env-j2aivp8_/overlay/lib/python3.8/site-packages/setuptools/dist.py", line 885, in parse_config_files
      pyprojecttoml.apply_configuration(self, filename, ignore_option_errors)
    File "/tmp/pip-build-env-j2aivp8_/overlay/lib/python3.8/site-packages/setuptools/config/pyprojecttoml.py", line 62, in apply_configuration
      config = read_configuration(filepath, True, ignore_option_errors, dist)
    File "/tmp/pip-build-env-j2aivp8_/overlay/lib/python3.8/site-packages/setuptools/config/pyprojecttoml.py", line 126, in read_configuration
      validate(subset, filepath)
    File "/tmp/pip-build-env-j2aivp8_/overlay/lib/python3.8/site-packages/setuptools/config/pyprojecttoml.py", line 51, in validate
      raise ValueError(f"{error}\n{summary}") from None
  ValueError: invalid pyproject.toml config: `project.dependencies[0]`.
  configuration error: `project.dependencies[0]` must be pep508
  ----------------------------------------
ERROR: Command errored out with exit status 1: /usr/bin/python3 /tmp/tmp6s9_fa0q get_requires_for_build_wheel /tmp/tmptw66vuzk Check the logs for full command output.```

**Comment by Ristellise** [comment]
Hi! Thanks for bringing it up. I will fix it in a bit.

**Comment by Ristellise** [comment]
Should be fixed as of Ristellise/RedGalaxy@4258602c48895e4b1b7e352479a44a3322c3722d
